packages/components-native/src/utils/test/wait.ts

```diff
@@ -0,0 +1,52 @@
+import { screen } from "@testing-library/react-native";
+import { act } from "react-test-renderer";
+
+/**
```

Comment:
This file has a lot of good comments but I'm looking for opinions about bringing these into Atlantis...